### DISCUSSION TITLE
RapidTableDetection项目 onnxruntime 1.16.3 用不了

### DISCUSSION THREAD

**Discussion by qq2499017550** [RapidAI/TableStructureRec#94] [Q&A]
onnxruntime.capi.onnxruntime_pybind11_state.Fail: [ONNXRuntimeError] : 1 : FAIL : Load model from /home/zhangjie/miniconda3/envs/py310/lib/python3.10/site-packages/rapid_table_det/models/yolo_obj_det.onnx failed:/onnxruntime_src/onnxruntime/core/graph/model.cc:149 onnxruntime::Model::Model(onnx::ModelProto&&, const PathString&, const IOnnxRuntimeOpSchemaRegistryList*, const onnxruntime::logging::Logger&, const onnxruntime::ModelOptions&) Unsupported model IR version: 10, max supported IR version: 9

**Comment by Joker1212** [comment]
需要升级一下onnxruntime，因为模型转换的opset版本不支持了
